test/logflare_web/controllers/api/team_controller_test.exs

```diff
@@ -201,4 +203,100 @@ defmodule LogflareWeb.Api.TeamControllerTest do
              |> assert_schema("NotFoundResponse") == %{error: "Not Found"}
     end
   end
+
+  describe "add_member/2" do
+    test "adds an existing user to a team", %{
```

Comment:
also needs case where there is no existing user record and we only create the record on `team_users` table